src/sales-channels/asc/amazon-orders-all.md

```diff
@@ -0,0 +1,77 @@
+---
+title: View Amazon Orders
+---
+
+
+There are 2 ways to view your Amazon orders, _Recent Orders_ and _All Orders_.
+
+Both options show you basic order information, as received from Amazon, including:
+
+- Purchase Date
+- Order Number
+- Status
+- Buyer's Name
+- Grand Total
+- Order Notes
+
+_All Orders_ view adds filtering options for order searches.
+
+{:.bs-callout .bs-callout-info}
+With the exception of the _Order Notes_ column, the _Amazon orders_ table is populated with order information as received from Amazon. The _Order Notes_ column is updated by Magento as the order processes.
+
+## Recent orders
+
+You can view your most recent orders in the _Recent Orders_ section of the [store dashboard]({% link sales-channels/asc/amazon-store-dashboard.md %}).
+
+![]({% link sales-channels/asc/assets/amazon-recent-orders-imported.png %}){: .zoom}
+_Recent Orders_
+
+### View recent Amazon orders
+
+1. Click **View Store** on a store card.
+
+1. View your orders in the _Recent Orders_ section.
+
+1. To view order details, click the Amazon order number in the _Order Number_ column.
+
+   Depending on your [order import setting]({% link sales-channels/asc/order-settings.md %}), the order details will open in Amazon or in the [Magento orders workflow]({% link sales/orders.md %}).
+
+## View all orders
+
+You can view all of your Amazon orders on the _Amazon orders_ page (also referred to as the _All Orders_ view). The Amazon Orders table is similar to the _Recent Orders_ section of the store dashboard, but allows you to view all of your Amazon orders and to narrow your orders list with the following filter options:
+
+- Purchase Date (range)
+- Order Number
+- Buyer's Name
+- Total (range)
+- Status
+
+![]({% link sales-channels/asc/assets/amazon-orders-list-all.png %}){: .zoom}
+_Amazon orders_
+
+### View all Amazon orders
+
+1. Click **View Store** on a store card.
+
+1. Click **All Orders** in the _Recent Orders_ section.
+
+1. To narrow the list or search for a specific order number, complete the **Filter by** parameters and click **Apply Filters**.
+
+1. To view order details, click the Amazon order number in the _Order Number_ column.
+
+   Depending on your [order import setting]({% link sales-channels/asc/order-settings.md %}), the order details will open in Amazon or in the [Magento orders workflow]({% link sales/orders.md %}).
+
+{:.bs-callout .bs-callout-tip}
+A few notes about the orders view:<br/>- If you have multiple Amazon store integrations, you may need to refresh your page view when switching between store views. This updates both the orders list and the pagination views for the current store.<br/>- When sorting by column, the sort only applies to the current view orders list. This will be addressed in a future release.<br/>- Depending on the width of your view window, you may see overlapping text in the columns. Widen your window view to expand the columns for the text to wrap.
```

Comment:
We are, unfortunately, forced to use <br/> in table cells but should avoid elsewhere. For a multi-line note, use `<div class="bs-callout-info" markdown="1">` and then format as you usually would.

Reply:
fixed in follow up commit